cloud/aws/fs_aws.cc

```diff
@@ -172,17 +172,17 @@ Status AwsCloudAccessCredentials::GetCredentialsProvider(
 // The AWS credentials are specified to the constructor via
 // access_key_id and secret_key.
 //
-AwsEnv::AwsEnv(const std::shared_ptr<FileSystem>& underlying_fs,
-               const CloudEnvOptions& _cloud_env_options,
-               const std::shared_ptr<Logger>& info_log)
-    : CloudEnvImpl(_cloud_env_options, underlying_fs, info_log) {
+AwsFileSystem::AwsFileSystem(const std::shared_ptr<FileSystem>& underlying_fs,
```

Comment:
If the class name is `AwsFileSystem` I would expect the filename to be `aws_fs.cc`, not `fs_aws.cc`.